packages/lesswrong/server/notificationCallbacks.tsx

```diff
@@ -126,19 +127,44 @@ export async function postsNewNotifications (post: DbPost) {
 
 postPublishedCallback.add(postsNewNotifications);
 
-getCollectionHooks("Posts").updateAsync.add(async function PostsEditMeetupNotifications ({document: newPost, oldDocument: oldPost}: {document: DbPost, oldDocument: DbPost}) {
+getCollectionHooks("Posts").updateAsync.add(async function eventUpdatedNotifications ({document: newPost, oldDocument: oldPost}: {document: DbPost, oldDocument: DbPost}) {
+  // don't bother notifying people about past or unscheduled events
+  const isUpcomingEvent = newPost.startTime && moment().isBefore(moment(newPost.startTime))
+  // only send notifications if the event was already published *before* being edited
+  const alreadyPublished = !oldPost.draft && !newPost.draft && !oldPost.authorIsUnreviewed && !newPost.authorIsUnreviewed
   if (
     (
       !_.isEqual(newPost.mongoLocation, oldPost.mongoLocation) ||
+      (newPost.onlineEvent !== oldPost.onlineEvent) ||
       (newPost.startTime !== oldPost.startTime) || 
       (newPost.endTime !== oldPost.endTime)
     )
-    && newPost.mongoLocation && newPost.isEvent && !newPost.draft && !newPost.authorIsUnreviewed)
+    && newPost.mongoLocation && newPost.isEvent && isUpcomingEvent && alreadyPublished)
   {
-    const usersToNotify = await getUsersWhereLocationIsInNotificationRadius(newPost.mongoLocation)
-    const userIds = usersToNotify.map(user => user._id)
-    const usersIdsWithoutAuthor = userIds.filter(id => id !== newPost.userId)
```

Comment:
Since events can be edited by users other than the author (i.e. admins and other group organizers), I decided to stop excluding them